bankcraft/agent/employer.py

```diff
@@ -7,14 +8,24 @@
 class Employer(GeneralAgent):
     def __init__(self, model):
         super().__init__(model)
-        self.pay_period = random.choice([steps['biweekly'], steps['month']])
-        self.employees = []
+        self.pay_period = random.choice([steps['biweekly']])
+        self._num_pays_per_year = steps['year'] // self.pay_period
+
+        self.employees = [] # list of [person, salary, salary_per_pay]
```

Comment:
Let's change this data structure to a list of dictionaries (instead of list of lists). It will help make the code more readable. See comments below.